mkdocs/topics/testing.md

```diff
@@ -0,0 +1,56 @@
+# Testing
```

Comment:
@tgsoverly here's some aspiration docs for tests. I used syntax for the trace references that was along the lines of what we discussed yesterday.